### PR TITLE
feat(screen): add style to home screen

### LINKED ISSUE #3: Home Screen Style
# REQUIREMENTS
Create base style for Home Screen

# DESIGN

### PR DESCRIPTION
**Pull request by aguirre-valeria**
# Description
 - Add style to home screen
 
 # Testing
I have used the react-native components. It's the skeleton only, to add icons later.
 
  ## Screenshots 

a

### CODE REVIEW

**Review by Charly77955 — COMMENTED** [review]
Check

**Review by Ch4s3d — APPROVED** [review]

**Review by Charly77955 — APPROVED** [review]
ok

File: ReservaVuelos/components/MyFlightsScreen/MyFlightsScreen.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                </View>
+                <View style={MyFlightsStyles.lineEnd}></View>
+
+             </TouchableOpacity>
+            <View style={MyFlightsStyles.containerAdd}>
+                <Text>+</Text>
+            </View>
+        </View>
+    )
+}
```
Comment (YilKanda) [inline ReservaVuelos/components/MyFlightsScreen/MyFlightsScreen.js]: Remove the extra space.

**Review by YilKanda — APPROVED** [review]

File: ReservaVuelos/components/MyFlightsScreen/MyFlightsScreen.js (hunk 1 above)
Comment (aguirre-valeria) [reply ReservaVuelos/components/MyFlightsScreen/MyFlightsScreen.js]: Extra space removed.

**Pull request merged**